fees/assetchain.ts

```diff
@@ -0,0 +1,45 @@
+import { Adapter, FetchOptions, ProtocolType } from "../adapters/types";
+import { CHAIN } from "../helpers/chains";
+import { httpGet } from "../utils/fetchURL";
+
+const contractAddress = '0xFC00FACE00000000000000000000000000000000';
+
+
+async function getFees24Hr(api: any) {
+  const currentEpoch = await api.call({ abi: 'uint256:currentSealedEpoch', target: contractAddress })
+  const snapshotABI = 'function getEpochSnapshot(uint256 epoch) view returns (uint256 endTime, uint256 epochFee, uint256 totalBaseRewardWeight, uint256 totalTxRewardWeight, uint256 baseRewardPerSecond, uint256 totalStake, uint256 totalSupply)'
+
+  // Calculate how many 4-hour epochs are in 24 hours (6 epochs)
+  const epochsToFetch = 6;
+  const epochs: any = [];
+
+  // Create array of epoch numbers to fetch (current and previous 5)
+  for (let i = 0; i < epochsToFetch; i++) epochs.push(+currentEpoch - i);
+  const epochData = await api.multiCall({ abi: snapshotABI, calls: epochs, target: contractAddress })
+  return epochData.reduce((acc: any, data: any) => acc + +data.epochFee, 0)
+}
+
+export async function getPrice() {
```

Comment:
the PR is mostly good except for this PR, I cant find the token on coingecko, is it traded in some exchange atm?

Reply:
Yes, it is.
There is a coinmarketcap page here for the token.
https://coinmarketcap.com/currencies/xend-finance/

Reply:
Hello @g1nt0ki
does this dispell your concerns?
can we proceed with merging the pr pls
Thanks